lib/zipkin-tracer/rack/zipkin-tracer.rb

```diff
@@ -94,7 +94,8 @@ def synchronize(&block)
       @lock.synchronize do
         yield
       end
-    rescue# Nothing wonky that the tracer does should stop us from using the app!!!
+    rescue StandardError => e # Nothing wonky that the tracer does should stop us from using the app!!!
+      @config.logger.error("Exception #{e.message} while sending Zipkin traces. #{e.backtrace}")
```

Comment:
should this be error level? if tracer shouldn't stop us from using the app, and we have a problem that issues an error, could the amount of errors take down the app (ex. fill-up log files)